packages/vite-plugin-react-docgen-typescript/tsconfig.json

```diff
@@ -13,6 +13,7 @@
     "sourceMap": true,
     "target": "es2018",
     "jsx": "react",
-    "strictNullChecks": true
+    "strictNullChecks": true,
```

Comment:
While you're at it, might want to change `moduleResolution` to `bundler`.  `node` only supports the features from node10 and below (so you don't get package.json exports maps, for instance).